### PR TITLE
Add App Catalogue management

### PR DESCRIPTION
**Pull request by ouranos**
Add App Catalogue management:

image

image

## Dependencies:
maestrano/mno-enterprise#448
maestrano/maestrano-hub#662

### CODE REVIEW

File: src/app/components/mnoe-api/admin/apps.svc.coffee [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  _self = @
+
+  appsPromise = null
+
+  @list = ->
+    return appsPromise if appsPromise?
+    appsPromise = MnoeAdminApiSvc.all('apps').getList().then(
+      (response) ->
+        response
+      (error) ->
```
Comment (alexnoox) [inline src/app/components/mnoe-api/admin/apps.svc.coffee]: If you only need the error, use `catch`

File: src/app/views/settings/apps/apps.controller.coffee [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    )
+    modalInstance.result.then(
+      (apps) ->
+        MnoeApps.enableMultiple(_.map(apps, 'id')).then(
+          ->
+            vm.enabledApps = vm.enabledApps.concat(apps)
+        )
+    )
+
+
```
Comment (alexnoox) [inline src/app/views/settings/apps/apps.controller.coffee]: Remove double empty line

File: src/app/views/settings/apps/apps.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<div>
+  <div class="row">
+    <div class="col-xs-12">
+      <!--<h1 translate>mnoe_admin_panel.dashboard.organization.organization_info</h1>-->
```
Comment (alexnoox) [inline src/app/views/settings/apps/apps.html]: Remove commented code

**Comment by ouranos** [comment]
@alexnoox code review applied and added style
Updated the screenshot

**Comment by alexnoox** [comment]
noice

**Pull request merged**